robotiq_gazebo/config/gripper_controller.yaml

```diff
@@ -0,0 +1,6 @@
+gazebo_gripper:
```

Comment:
Will this work with multiple grippers in the same simulation? What needs to be namespaced or prefixed?

Reply:
I have added a controller in the newest commit and renamed them both with the prefix indicating the arms they are attached to. A separate controller is needed for each gripper.